### ISSUE TITLE
How to implement?

### ISSUE THREAD

**Issue by MinhThanghus** [seareale/BboxCutMix#1]
I have folder include image and label. How to generate data ?

**Comment by seareale** [comment]
I used YOLO type in bbox labels. If you want to have a easy way, use this [code](https://github.com/ultralytics/yolov5/blob/master/utils/dataloaders.py)(in YOLOv5 repo) and also check [this](https://github.com/seareale/BboxCutMix#how-to-use).
 
BboxCutmix uses all images in the train dataset. So if you don't use this code in 'dataloader'(e.g. like using data augmentation after loading a batch), there could be a overfitting.

**Comment by MinhThanghus** [comment]
thank you, i will try
Your idea is great
